regcomp.h

```diff
@@ -150,7 +186,12 @@ struct regnode_p {
     U8	flags;
     U8  type;
     U16 next_off;
-    SV * arg1;
+#if PTRSIZE == 8
```

Comment:
The comment to `struct regnode_p` just above could also mention the caveats around 64-bit pointers.

Reply:
+1